tests/UI/campaigns/functional/FO/03_userAccount/merchandisereturns/01_consultReturnDetails.js

```diff
@@ -0,0 +1,456 @@
+
+        // Change FO language
+        await homePage.changeLanguage(page, 'en');
+
+        const isHomePage = await homePage.isHomePage(page);
+        await expect(isHomePage, 'Home page is not displayed').to.be.true;
+      });
+
+      it('should go to my account page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToAccountPage2', baseContext);
+
+        await homePage.goToMyAccountPage(page);
+
+        const pageTitle = await myAccountPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(myAccountPage.pageTitle);
+      });
+
+      it('should go to \'Order history and details\' page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'goToOrderHistoryPage', baseContext);
+
+        await myAccountPage.goToHistoryAndDetailsPage(page);
+
+        const pageTitle = await orderHistoryPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(orderHistoryPage.pageTitle);
+      });
+
+      it('should go to the first order in the list and check the existence of order return form', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'isOrderReturnFormVisible', baseContext);
+
+        await orderHistoryPage.goToDetailsPage(page, 1);
+
+        const result = await orderDetailsPage.isOrderReturnFormVisible(page);
+        await expect(result).to.be.true;
+      });
+
+      it('should create a merchandise return', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'createMerchandiseReturn', baseContext);
+
+        await orderDetailsPage.requestMerchandiseReturn(page, 'message test');
+
+        const pageTitle = await foMerchandiseReturnsPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(foMerchandiseReturnsPage.pageTitle);
+      });
+    });
+
+    describe('Check merchandise returns', async () => {
+      it('should verify the Order reference', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'checkOrderReturnReference', baseContext);
+
+        const packageStatus = await foMerchandiseReturnsPage.getTextColumn(page, 'orderReference');
+        await expect(packageStatus).to.equal(orderReference);
+      });
+
+      it('should verify the Order return file name', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'checkOrderReturnFileName', baseContext);
+
+        const packageStatus = await foMerchandiseReturnsPage.getTextColumn(page, 'fileName');
+        await expect(packageStatus).to.contains('#RE00');
+      });
+
+      it('should verify the order return status', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', 'checkOrderReturnStatus', baseContext);
+
+        const packageStatus = await foMerchandiseReturnsPage.getTextColumn(page, 'status');
+        await expect(packageStatus).to.equal('Waiting for confirmation');
```

Comment:
Could you move the label to the `foMerchandiseReturnsPage` ?

Reply:
Done thx :+1: